src/tools/arakoon_url.ml

```diff
@@ -0,0 +1,47 @@
+(*
+Copyright 2016 iNuron NV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*)
+
+type url =
+  File of string
+  | Etcd of (((string * int) list) * string)
+
+let to_string = function
+  | File f -> f
+  | _ -> "??"
```

Comment:
Shouldn't be too hard to give a good string representation here?
sth a la `etcd://host1:port1,host2:port2/key`